client/src/hooks/helpers/useGuilds.tsx

```diff
@@ -49,13 +50,35 @@ export type GuildFromPlayerAddress = {
 export const useGuilds = () => {
   const {
     setup: {
-      components: { Guild, GuildMember, GuildWhitelist, Owner },
+      components: { Guild, GuildMember, GuildWhitelist, Owner, AddressName },
     },
   } = useDojo();
 
   const { getEntityName } = getEntitiesUtils();
   const { getAddressName } = useRealm();
 
+  const getPlayerList = (guild_entity_id: ID) => {
+    const players = Array.from(runQuery([Has(AddressName)])).map((playerEntity) => {
+      const player = getComponentValue(AddressName, playerEntity);
+
+      const inviteEntity = Array.from(
```

Comment:
const isInvited = getComponentValue(GuildWhitelist, getEntityIdsByKeys([player?.address,  guild_entity_id]))?.is_whitelisted;

instead of runquery